helpers/ingredients.js

```diff
@@ -137,7 +137,7 @@ module.exports.decrementAmountForProduction = function(id, amount, formula, form
       }
       for (let lot of lotsConsumed) {
         var time = Date.now() - lot.timestamp;
-        await Freshness.updateReport(lot.ingID, lot.name, lot.amount, time);
+        await Freshness.updateFreshnessReport(lot.ingID, lot.name, lot.amount, time);
```

Comment:
@dy46 Any other name changes I'm unaware of?

Reply:
if you pulled from master it should be alright